tests/plugins/range.test.ts

```diff
@@ -434,6 +434,13 @@ describe("range plugin", () => {
         expect(m.getters.getRangeString(range, "tao")).toBe(`'${name}'!A1`);
       }
     );
+
+    test.each([["s1!A1:s1!A9"], ["s1!!!A1:s1!!!A9"]])(
+      "xc with more than one exclamation mark does not throw error",
+      (range) => {
+        expect(() => m.getters.getRangeFromSheetXC("s1", range)).not.toThrowError();
```

Comment:
Better to test the result rather than just that it doesn't throw

Reply:
@LucasLefevre When there are too many exclamation marks in the input, the input border will become red, which indicates something wrong in the input, which I think it's clear. 
The real problem here is in the current 16.0 implementation this will come with errors and the freezing or even crashed application, and users cannot edit the input (a lot of error dialogs pop up and no confirm button below the input) which is very annoying. 
For now I only deal with the error/crash, give the users possibility to change the inputs and don't care the results after parsing. I think we can create a new task for it?